src/helpers/clipboard/clipboard_helpers.ts

```diff
@@ -46,11 +46,9 @@ function splitZoneForPaste(selection: Zone, splitWidth: number, splitHeight: num
 /**
  * Compute the complete zones where to paste the current clipboard
  */
-export function getPasteZones<T>(target: Zone[], content: T[][]): Zone[] {
-  if (!content.length || !content[0].length) {
+export function getPasteZones<T>(target: Zone[], height: number, width: number): Zone[] {
```

Comment:
Because we actually only need the height & width of the content.

Reply:
But it makes the code everywhere else longer and more prone to errors.
`content.length, content[0].length` instead of just `content`

Also the change is unrelated to the commit.
=> revert this part :wink: